src/bindings/scala/api/src/main/scala/com/ctc/omega_edit/api/ViewportEvent.scala

```diff
@@ -28,7 +28,7 @@ object ViewportEvent extends IntEnum[ViewportEvent] {
   case object Undo extends ViewportEvent(4)
   case object Clear extends ViewportEvent(8)
   case object Transform extends ViewportEvent(16)
-  case object Updated extends ViewportEvent(32)
+  case object Modified extends ViewportEvent(32)
```

Comment:
Keep with present tense of the verb as the rest of them are.  It was my bad originally to say `updated` rather than `update` to keep with the proper verb tense of the others.

Reply:
All that to say, use Modify here instead of Modified.